src/locales/pt.json

```diff
@@ -0,0 +1,136 @@
+{
+  "menu": {
+    "superbacked": {
+      "about": "Sobre",
+      "hide": "Ocultar",
+      "quit": "Sair"
+    },
+    "file": {
+      "file": "Arquivo",
+      "create": "Criar",
+      "duplicate": "Duplicar",
+      "restore": "Restaurar",
+      "newWindow": "Nova janela",
+      "closeWindow": "Fechar janela"
+    },
+    "edit": {
+      "edit": "Editar",
+      "undo": "Desfazer",
+      "redo": "Refazer",
+      "cut": "Cortar",
+      "copy": "Copiar",
+      "paste": "Colar",
+      "selectAll": "Selecionar tudo"
+    },
+    "insert": {
+      "insert": "Inserir",
+      "mnemonic": "Mnemônico BIP39",
+      "passphrase": "Senha de recuperação EFF",
+      "scanQrCode": "Escanear código QR"
```

Comment:
I would suggest "Ler código QR" instead of "Escanear". The word "scan" is used everywhere in software in english, but in portuguese, in the context of QR code, almost, if not all, software use "ler" instead.